stardis/model/base.py

```diff
@@ -110,4 +115,6 @@ def read_marcs_to_fv(fpath, atom_data, final_atomic_number=30):
         marcs_abundances_all[i] = marcs_abundances["mass_abundance"]
     marcs_abundances_all = marcs_abundances_all[:final_atomic_number]
 
-    return StellarModel(marcs_model_fv, marcs_abundances_all, boundary_temps, geometry)
+    return StellarModel(
```

Comment:
this class is getting resturctured later right?

Reply:
Yes. The current plan is to add the different attributes to carry along the information in the way that we actually want, and then switch over and get rid of the old redundant attributes later once the code can run without them (specifically marcs_model_fv, marcs_abundanes_all, and boundary_temps).